IPython/html/static/widgets/js/widget.js

```diff
@@ -558,7 +611,7 @@ define(["widgets/js/manager",
             /**
              * Makes browser interpret a numerical string as a pixel value.
              */
-            if (/^\d+\.?(\d+)?$/.test(value.trim())) {
+            if (value && /^\d+\.?(\d+)?$/.test(value.trim())) {
```

Comment:
`null`, `undefined`, and `string` values all work fine, but a `number` value raises an `undefined` error.  I opened a PR against your branch that fixes this https://github.com/jasongrout/ipython/pull/8.